datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -1391,14 +1368,55 @@ private void handleLeaderDoAssignment(boolean cleanUpOrphanNodes) {
 
     // schedule retry if failure
     if (!succeeded && !_leaderDoAssignmentScheduled.get()) {
-      _log.info("Schedule retry for leader assigning tasks");
-      _metrics.updateKeyedMeter(CoordinatorMetrics.KeyedMeter.HANDLE_LEADER_DO_ASSIGNMENT_NUM_RETRIES, 1);
-      _leaderDoAssignmentScheduled.set(true);
-      _leaderDoAssignmentScheduledFuture = _scheduledExecutor.schedule(() -> {
-        _eventQueue.put(CoordinatorEvent.createLeaderDoAssignmentEvent(cleanUpOrphanNodes));
-        _leaderDoAssignmentScheduled.set(false);
-      }, _config.getRetryIntervalMs(), TimeUnit.MILLISECONDS);
+      scheduleLeaderDoAssignmentRetry(cleanUpOrphanNodes);
+    }
+  }
+
+  void scheduleLeaderDoAssignmentRetry(boolean cleanUpOrphanNodes) {
```

Comment:
You changed the variable name `cleanUpOrphanNodes` in #924

Reply:
I will merge those changes when #924 is checked in @vmaheshw

Reply:
Renamed the parameter